templates/k8s.yaml

```diff
@@ -91,14 +91,30 @@ provision:
     EOF
     systemctl restart containerd
 # See <https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/>
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    test -e /etc/kubernetes/tmp/pull-done && exit 0
+    systemctl stop kubelet
+    if [ -d /var/lib/kubernetes/images ]; then
```

Comment:
This needs a comment line

Reply:
Probably needs documentation.

Reply:
It could also be taken out of the template completely, since it doesn't do anything by default (is opt-in)

If you are adding a pull or a mount, you could also copy/paste the rest of the script (from somewhere).

Reply:
For k3s it is slightly different, since the airgap feature is supported by the upstream installer script.

So it doesn't change much for lima, to include features for it. `curl -sfL https://get.k3s.io | sh -`